# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../bcron/bcron-0.10-no-python.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Do not use python for generating 'conf_sendmail.c' from 'conf-sendmail'

--- a/Makefile	2013-04-18 13:32:07.380724339 +0200
+++ b/Makefile	2013-04-18 13:32:23.460768578 +0200
@@ -90,7 +90,9 @@
 	chmod 755 compile
 
 conf_sendmail.c: conf-sendmail
-	python -c 'import shlex; line=shlex.split(open("conf-sendmail").readline().strip()); print "const char* sendmail[] = { "+",".join(["\"%s\""%a for a in line])+",NULL };"' >conf_sendmail.c
+	( confsendmail=`head -n 1 conf-sendmail | tr -s ' '`; \
+	  printf "const char* sendmail[] = { `for x in $${confsendmail}; do printf '\"%s\",' $${x}; done`NULL };\n" \
+	) >conf_sendmail.c
 
 connection.o: compile connection.c bcron.h
 	./compile connection.c
